utils/objects/scene_object.py

```diff
@@ -1,22 +1,29 @@
 
+import json
 from typing import List
 
-
 class SceneObject:
-    def __init__(self, actions:List[str], objetcs:List[str],
-                 description:str, text_in_scene:List[str],
+    def __init__(self, actions:List[str]=[], objetcs:List[str]=[],
+                 description:str=[], text_in_scene:List[str]=[],
                  duration:int=0,scene_start:int=0,
+                 title:str="",
```

Comment:
We might also just make the tasks a class itself that this one can then borrow from so we dont have the same rewriting problem as above.